src/styles/blocks/_catalog.scss

```diff
@@ -0,0 +1,35 @@
+.catalog {
+  display: grid;
+  justify-content: center;
+  grid-template-columns: repeat(4, $cardWidth);
+  grid-column-gap: 48px;
+
+  margin: 0 auto;
+
+  @media (max-width: 1024px) {
+    grid-template-columns: repeat(3, $cardWidth);
+  }
+
+  @media (max-width: 768px) {
+    grid-template-columns: repeat(2, $cardWidth);
+  }
+
+  @media (max-width: 488px) {
+    grid-template-columns: $cardWidth;
+  }
```

Comment:
There should be two columns **starting** at 488px
![image](https://user-images.githubusercontent.com/37161315/195093148-fe11a9ad-ac16-46dd-afbf-20e8fa4e3a7a.png)